Failure to compile with Rust 1.75

When trying to use the new version with the Rust 1.75 compiler, I get the following error:
```
error[E0658]: use of unstable library feature 'error_in_core'
  --> arbitrary-1.4.0/src/lib.rs:52:6
   |
52 | impl core::error::Error for MaxRecursionReached {}
```

It looks like while `std::error::Error` has been around for a while, its presence in `core` is new as of Rust 1.81

--- a/.github/workflows/rust.yml
+++ b/.github/workflows/rust.yml
@@ -25,6 +25,14 @@ jobs:
       run: cargo test --verbose --all-features --all
     - name: Build Examples
       run: cargo build --examples --all-features --all
+  msrv:
+    name: Minimum Supported Rust Version
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - run: rustup toolchain add 1.63
+    - name: Build
+      run: cargo +1.63 check --lib --all-features
   clippy:
     name: Clippy
     runs-on: ubuntu-latest

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "arbitrary"
-version = "1.4.0" # Make sure this matches the derive crate version (not including the patch version)
+version = "1.4.1" # Make sure this matches the derive crate version (not including the patch version)
 authors = [
     "The Rust-Fuzz Project Developers",
     "Nick Fitzgerald <fitzgen@gmail.com>",

--- a/src/foreign/alloc/borrow.rs
+++ b/src/foreign/alloc/borrow.rs
@@ -5,7 +5,7 @@ use {
 
 impl<'a, A> Arbitrary<'a> for Cow<'a, A>
 where
-    A: ToOwned + ?Sized,
+    A: ToOwned + ?Sized + 'a,
     <A as ToOwned>::Owned: Arbitrary<'a>,
 {
     fn arbitrary(u: &mut Unstructured<'a>) -> Result<Self> {

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -49,7 +49,7 @@ impl core::fmt::Display for MaxRecursionReached {
     }
 }
 
-impl core::error::Error for MaxRecursionReached {}
+impl std::error::Error for MaxRecursionReached {}
 
 /// Generate arbitrary structured values from raw, unstructured data.
 ///

